configure

```diff
@@ -24,7 +24,7 @@ if test -n "${ZSH_VERSION+set}" && (emulate sh) >/dev/null 2>&1; then :
   alias -g '${1+"$@"}'='"$@"'
   setopt NO_GLOB_SUBST
 else
-  case `(set -o) 2>/dev/null` in #(
+  case $( (set -o) 2>/dev/null) in #(
```

Comment:
`configure` is autogenerated. The fact that `autoconf` uses backticks is further evidence that there is no real deprecation.